apps/portal/src/app/connect/pay/faqs/page.mdx

```diff
@@ -29,6 +29,19 @@ This can include intents, native bridges, third party bridges, cross-chain messa
 health, price, and speed to guarantee the best path every time. 
 </Details>
 
+<Details summary="Am I able to use Universal Bridge with other wallet infrastructure providers?"> 
+Yes, Universal Bridge is designed to be agnostic to wallet infrastructure. You can use it with any wallet service. 
+
+We have adapters to easily integrate the following solutions:
```

Comment:
Don't even need the adapters actually. UB returns raw transaction data that any wallet should accept